demisto_sdk/commands/common/tools.py

```diff
@@ -731,3 +745,61 @@ def is_file_from_content_repo(file_path: str) -> Tuple[bool, str]:
         return True, '/'.join(input_path_parts[len(content_path_parts):])
     else:
         return False, ''
+
+
+class FileFinder:
```

Comment:
this class kind of breaks the standard of this file, and its not like actually its needed, lets stick to single functions and drop it